acceptance/repository-postgresql/src/__tests__/postgresql.datasource.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/test-repository-postgresql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CrudFeatures, DataSourceOptions} from '@loopback/repository-tests';
+
+const connector = require('loopback-connector-postgresql');
+
+export const POSTGRESQL_CONFIG: DataSourceOptions = {
+  connector,
+  host: process.env.POSTGRESQL_HOST || 'localhost',
+  port: process.env.POSTGRESQL_PORT || 5432,
+  database: process.env.POSTGRESQL_DATABASE || 'testdb',
```

Comment:
Please use `repository-tests`, the same value we use in other LB4 repository-acceptance tests.

Reply:
According to the [documentation](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS), it can't include a `-` so I'll use `repository_tests`. Should I modify the others to be consistent with this one?

Reply:
> I'll use `repository_tests`. 

👍 

> Should I modify the others to be consistent with this one?

I don't mind either way, please decide yourself.